resolwe_bio/tests/processes/test_upload.py

```diff
@@ -52,6 +52,9 @@ def test_bam_upload(self):
     @with_resolwe_host
     @tag_process("upload-expression")
     def test_upload_expression(self):
```

Comment:
Extend the test with additional uploads of files with alternative extentions (txt.gz, ..) to test if the renaming works as expected.